charts/create-kafka-queues/values.yaml

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-exports: {}
+exports:
   # zookeeper connection url
-  # zookeeper: palisade-zookeeper:2181
+  zookeeper: palisade-zookeeper:2181
```

Comment:
Seems a bit dodgy hard-coding the port in here, or even the zookeeper name. Can we not compose these in a _helpers template function using the values provided in the origin charts?